### DISCUSSION TITLE
Documentation for installation

### DISCUSSION THREAD

**Discussion by monet820** [aquasecurity/trivy#5369] [Documentation]
### Description

I was just wondering if the winget package, is supported by Trivy and if it should be noted in the readme or the [install-using-package-manager](https://aquasecurity.github.io/trivy/v0.45/getting-started/installation/#install-using-package-manager) ?

winget search trivy

Name  Id                 Version Source        
----------------------------------------       
Trivy AquaSecurity.Trivy 0.45.1  winget        

winget install Trivy
Found Trivy [AquaSecurity.Trivy] Version 0.45.1
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/aquasecurity/trivy/releases/download/v0.45.1/trivy_0.45.1_windows-64bit.zip
  ██████████████████████████████  49.6 MB / 49.6 MB
Successfully verified installer hash
Extracting archive...
Successfully extracted archive
Starting package install...
Path environment variable modified; restart your shell to use the new value.
Command line alias added: "trivy"
Successfully installed

### Suggestions

winget install Trivy should be mentioned in how to install Trivy on a windows machine.

**Comment by knqyf263** [comment]
Sounds like a good idea.
